be/src/exec/vectorized/sorting/sort_column.cpp

```diff
@@ -138,11 +138,6 @@ class VerticalColumnSorter final : public ColumnVisitorAdapter<VerticalColumnSor
     size_t get_limited() const { return _pruned_limit; }
 
     Status do_visit(const vectorized::NullableColumn& column) {
-        // Fastpath
-        if (!column.has_null()) {
```

Comment:
why remove the fast path?

Reply:
This fastpath is wrong, vertical sort for multiple columns has no fastpath.